moped-editor/src/views/projects/projectView/ProjectNotes.js

```diff
@@ -316,6 +325,16 @@ const ProjectNotes = ({
     setDeleteConfirmationId(id);
   };
 
+  const handleECaprisSwitch = () => {
+    updateShouldSyncECapris({
+      variables: {
+        projectId: noteProjectId,
+        shouldSync: !shouldSyncFromECapris,
+      },
+    });
```

Comment:
Didn't see that `handleSnackbar` was readily available in this file so it could be nice to handle success and errors here like in `submitEditNote`. What do you think about adding?